### ISSUE TITLE
Linker errors in TS_NJKWebViewProgress

### ISSUE THREAD

**Issue by krasnorutsky** [TapSense/tapsense-ads-ios-sdk#2]
We found that you use NJKWebViewProgress class in you code.

https://github.com/ninjinkun/NJKWebViewProgress/blob/master/NJKWebViewProgress/NJKWebViewProgress.m

In your code this class is renamed into TS_NJKWebViewProgress to prevent collisions. But the global constant completeRPCURLPath (from this class) is not renamed. So when we try to use other third-party libraries in our code - We get linking errors.

Could you please rename the completeRPCURLPath constant in your code ?

**Comment by willchen0506** [comment]
Thanks for addressing this. We will release an update soon.

**Comment by willchen0506** [comment]
```
duplicate symbol _completeRPCURLPath in:
    /Users/Will/Library/Developer/Xcode/DerivedData/InterstitialAdSample-cfpooxcmjabpaufxlbyrlnhaadlo/Build/Intermediates/InterstitialAdSample.build/Debug-iphonesimulator/InterstitialAdSample.build/Objects-normal/x86_64/NJKWebViewProgress.o
    ../../TapSenseAds/TapSenseAds.embeddedframework/TapSenseAds.framework/TapSenseAds(TS_NJKWebViewProgress.o)
ld: 1 duplicate symbol for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by willchen0506** [comment]
This has been fixed in the latest release (3.2.2). Thanks again for pointing it out.
